lib/config/config-loader.js

```diff
@@ -173,7 +173,7 @@ async function loadConfigFile(filePath, allowTS) {
      */
     if (allowTS && isTS && !isDeno && !isBun) {
 
-        const createJiti = await import("jiti").then(jitiModule => jitiModule.default, () => {
+        const createJiti = await import("jiti").then(jitiModule => (typeof jitiModule?.createJiti === "function" ? jitiModule.createJiti : jitiModule.default), () => {
```

Comment:
This will not throw an error if jiti v1.26 is installed, correct?

Reply:
Nope, just checked it's fine.